ecl/key/AccuracyTestModified.xml

```diff
@@ -0,0 +1,11 @@
+ <Warning><Code>6003</Code><Filename>/home/nsel1727/.HPCCSystems/bundles/_versions/DBSCAN/V1_0_0/DBSCAN/internal/globalMerge.ecl</Filename><Line>158</Line><Source>eclcc</Source><Message>pointer of type &#226;&#128;&#152;void *&#226;&#128;&#153; used in arithmetic [-Wpointer-arith]</Message></Warning>
+ <Warning><Code>6003</Code><Filename>/home/nsel1727/.HPCCSystems/bundles/_versions/DBSCAN/V1_0_0/DBSCAN/internal/globalMerge.ecl</Filename><Line>160</Line><Source>eclcc</Source><Message>pointer of type &#226;&#128;&#152;void *&#226;&#128;&#153; used in arithmetic [-Wpointer-arith]</Message></Warning>
+ <Warning><Code>6003</Code><Filename>/home/nsel1727/.HPCCSystems/bundles/_versions/DBSCAN/V1_0_0/DBSCAN/internal/globalMerge.ecl</Filename><Line>162</Line><Source>eclcc</Source><Message>pointer of type &#226;&#128;&#152;void *&#226;&#128;&#153; used in arithmetic [-Wpointer-arith]</Message></Warning>
+ <Warning><Code>6003</Code><Filename>/home/nsel1727/.HPCCSystems/bundles/_versions/DBSCAN/V1_0_0/DBSCAN/internal/globalMerge.ecl</Filename><Line>164</Line><Source>eclcc</Source><Message>pointer of type &#226;&#128;&#152;void *&#226;&#128;&#153; used in arithmetic [-Wpointer-arith]</Message></Warning>
+ <Exception><Code>3000</Code><Source>eclccserver</Source><Message>Compile/Link failed for W20220616-155205 (see W20220616-155205.cc.log for details)</Message></Exception>
```

Comment:
After the ML-495 will be merged, those errors won't appear again, please remove those 5  (Warning and Exception) lines.